client/my-sites/sites/sites.jsx

```diff
@@ -93,4 +108,13 @@ export default React.createClass( {
 			</Main>
 		);
 	}
-} );
+}
+
+export default connect(
+	( state ) => ( {
```

Comment:
An abstraction layer sounds like a neat idea to avoid duplication, and will do well to tease out potential issues with treating Redux sites state as the canonical source of truth. 👍